content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -186,8 +186,6 @@ There are some implicit conventions worth noting here:
 
 ### Cluster-level default constraints
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
-
```

Comment:
From `kubescheduler.config.k8s.io/v1beta1` I'm inferring that at least part of this is beta. What's the best way to explain to readers about whether or not this kind of setting is GA?

Reply:
In the sense that the configuration API is beta, then yes, this could be considered beta as well. @Huang-Wei do you agree?

Reply:
I'm fine with either. Actually the code quality for this part is GA, but users can still disable (which is the default case) it by not setting the arg in scheduler configuration (v1beta1).

Reply:
I put the annotation back (but beta)